src/constants.js

```diff
@@ -154,6 +154,13 @@ export const GOAL_STATUS = {
   CLOSED: 'Closed',
 };
 
+export const OBJECTIVE_STATUS = {
```

Comment:
Can we write a jira to dupe this in the frontend code and convert all the usage in the frontend to also use that?

Reply:
And it might make sense to convert the backend to use the constant in this PR, since it's only a handful of files, there's more in the frontend so that makes sense to break out separately. IMHO, anyways

Reply:
WIll do 👍

Reply:
Created: https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1035

Reply:
Ok the BE statuses should now be updated.